todo.js

```diff
@@ -0,0 +1,125 @@
+  myToDo = JSON.parse(localStorage.myToDo);
+}
+
+// Selectors
+const list = document.getElementById('to-do-list');
+const toDoButton = document.getElementById('submit-button');
+const toDoInput = document.getElementById('to-do-input');
+const toDoClear = document.getElementById('to-do-clear');
+
+// Functions
+function renderList() {
+  myToDo.forEach((todo) => {
+    // li .task-container
+    const newTodo = document.createElement('li');
+    newTodo.classList.add('task-container');
+    list.appendChild(newTodo);
+    // div .checkbox-description-container
+    const checkDiv = document.createElement('div');
+    checkDiv.classList.add('checkbox-description-container');
+    newTodo.appendChild(checkDiv);
+    // input-checkbox
+    const checkbox = document.createElement('input');
+    checkbox.classList.add('checkbox-input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = (`${todo.completed ? 'true' : ''}`);
+    checkDiv.appendChild(checkbox);
+    // input-description
+    const pDescription = document.createElement('input');
+    pDescription.classList.add(`description-${todo.index}`);
+    pDescription.value = todo.description;
+    pDescription.classList.add('edit-input');
+    pDescription.classList.add(`${todo.completed ? 'completed-task' : 'checkbox-description-container'}`);
+    checkDiv.appendChild(pDescription);
+    pDescription.addEventListener('keydown', (e) => {
+      const newDescription = pDescription.value;
+      if (e.keyCode === 13) {
+        const oldToDo = JSON.parse(localStorage.myToDo);
+        oldToDo[todo.index].description = newDescription;
+        localStorage.setItem('myToDo', JSON.stringify(oldToDo));
+      }
+    });
+    // div .buttons-container
+    const buttonsDiv = document.createElement('div');
+    buttonsDiv.classList.add('show');
+    newTodo.appendChild(buttonsDiv);
+    // div .edit-buttons-container
+    const editButtonsDiv = document.createElement('div');
+    editButtonsDiv.setAttribute('id', `edit-buttons-${todo.index}`);
+    editButtonsDiv.classList.add('hidden');
+    editButtonsDiv.classList.add(`buttons-container-${todo.index}`);
+    buttonsDiv.appendChild(editButtonsDiv);
+    // Button delete
+    const buttonDelete = document.createElement('button');
+    buttonDelete.classList.add('delete-button');
+    buttonDelete.innerHTML = '<i class="fas fa-trash-alt"></i>';
+    buttonDelete.onclick = function () { deleteTodo(todo.index); };
+    editButtonsDiv.appendChild(buttonDelete);
+    // button menu .ellipsis
+    const buttonMenu = document.createElement('button');
+    buttonMenu.classList.add(`move-button-${todo.index}`);
+    buttonMenu.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
+    buttonMenu.onclick = function () { showButtons(todo.index); };
+    buttonsDiv.appendChild(buttonMenu);
+  });
+}
+
+function updateLocalStorage() {
+  localStorage.myToDo = JSON.stringify(myToDo);
+  document.location.reload();
+  return false;
```

Comment:
In this function, you are always returning a boolean value, but I can't see where you are using this boolean value in your code, would it be better if you make this a void function?